.github/workflows.src/release.tpl.yml

```diff
@@ -12,15 +12,14 @@ jobs:
     steps:
     - name: Determine package version
       shell: bash
-      env:
-        BRANCH: "${{ github.event.client_payload.branch }}"
-      run: echo ::set-output name=version::${BRANCH#releases/}
+      run: echo ::set-output name=branch::${GITHUB_REF#refs/heads/}
```

Comment:
I think you meant to do this:

```
run: |
    echo ::set-output name=branch::${GITHUB_REF#refs/heads/}
    echo ::set-output name=version::${GITHUB_REF#refs/heads/releases/}
```

Reply:
Ha, I noticed while I was reading 😂

<img width="263" alt="Screenshot 2020-09-23 at 19 00 42" src="https://user-images.githubusercontent.com/55281/94044949-188a5d00-fdcf-11ea-8466-50a889db0b63.png">
<img width="633" alt="Screenshot 2020-09-23 at 19 00 52" src="https://user-images.githubusercontent.com/55281/94044966-1de7a780-fdcf-11ea-98e6-75d443d7cacc.png">